Gemfile

```diff
@@ -4,15 +4,17 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'pry-rails'
+group :development, :test do
+  gem 'pry-rails'
 
-gem 'activestorage', '~> 6.0'
-gem 'capybara', '~> 3.33'
-gem 'puma', '~> 4.3'
-gem 'rspec-rails'
-gem 'rspec_junit_formatter', '~> 0.4'
-gem 'rubocop', '~> 1.0'
-gem 'sassc', '~> 2.4'
-gem 'selenium-webdriver', '~> 3.142'
-gem 'sprockets-rails', '~> 3.2'
-gem 'sqlite3', '~> 1.4'
+  gem 'activestorage', '~> 6.0'
+  gem 'capybara', '~> 3.33'
+  gem 'puma', '~> 4.3'
+  gem 'rspec-rails'
+  gem 'rspec_junit_formatter', '~> 0.4'
+  gem 'rubocop', '~> 1.0'
+  gem 'sassc', '~> 2.4'
```

Comment:
I think that this line could be removed.

Reply:
should we remove this line of the one in the gemspec?